src/picosvg/svg_meta.py

```diff
@@ -166,7 +166,9 @@ def parse_css_declarations(
         if declaration.count(":") == 1:
             property_name, value = declaration.split(":")
             property_name = property_name.strip()
-            if property_names is None or property_name in property_names:
+            if (
+                property_names is None or property_name in property_names
+            ) and not property_name.startswith("-"):
```

Comment:
Hm, if `property_name in property_names` then we should include in the parsed output attributes, no matter if it starts or not with `-`. E.g. maybe in the future we might choose to support some of these private attributes, who know.
So instead of `and` you should use `or` for the last condition.

```python
            if property_names is None or property_name in property_names or not property_name.startswith("-"):
```

does that make sense?

Reply:
Just wondering, is there any limitation on why xml etree attribute should not starts with dash? Is it considered a bug or required by the XML standard?

Reply:
I honestly have no idea, but it's very likely given that lxml is such a popular library, in turn relying on the libxml2 C library. Maybe you could ask on lxml issue tracker.

Reply:
it turns out it's impossible for XML attribute names (or XML names in general) to start with a hyphen, see https://github.com/googlefonts/picosvg/issues/293#issuecomment-1562958015